AdminUI/Dockerfile.debug

```diff
@@ -0,0 +1,13 @@
+# Build stage
+FROM node:16-alpine as build-stage
+EXPOSE 4200
+
+WORKDIR /app
+
+# all configuration files are json
+COPY package*.json /app/
+COPY angular.json /app/
+COPY tsconfig*.json /app/
```

Comment:
As far as I can see, these lines are unnecessary. Because either you mount all the files from "/AdminUI/src" into the container (in this case, you have these three files anyway) or you don't mount them at all (in this case, you are missing all the other files). 
I would just remove these lines.

Reply:
Those files are on the root of the `AdminUI` folder, which contains the `src` folder as well. I would share `angular.json` and `tsconfig*.json` whith the host system the same way we do with the `src` folder, but it turns out that you can't have a volume that is not a directory.
Sharing the whole `AdminUI` is not viable because we'd be poluting the host system with compilation artifacts such as the `node_modules` folder.
Moreover, the `package*.json` files are necessary to build the container, which makes it absolutely mandatory that they're copied.

Reply:
Oh, yeah, I didn't see that you only mount the `src` folder.